.buildkite/pipeline.yml

```diff
@@ -79,35 +75,35 @@ steps:
               - "Makefile.debian10"
               - "Makefile.debian11"
               - "Makefile.debian12"
+        notify:
+          - github_commit_status:
+              context: "Release / Ubuntu X86_64"
 
-      - label: ":linux: Release / Ubuntu ARM - {{matrix.makefile}}"
+      - label: ":linux: Release / Ubuntu ARM - Makefile.debian9"
         key: "release-ubuntu-arm"
         command:
-          - ".buildkite/scripts/build.sh {{matrix.makefile}}"
-          - ".buildkite/scripts/publish.sh {{matrix.makefile}}"
-        branches: "^(main|\\d+\\.\\d+)$"
+          - ".buildkite/scripts/build.sh Makefile.debian9"
+          - ".buildkite/scripts/publish.sh Makefile.debian9"
+        if: build.branch == "main" || build.branch =~ /^[0-9]+\.[0-9]+$$/
         agents:
           provider: "aws"
           imagePrefix: "${IMAGE_UBUNTU_ARM_64}"
           instanceType: "t4g.large"
-        matrix:
-          setup:
-            makefile:
-              - "Makefile"
-              - "Makefile.debian7"
-              - "Makefile.debian8"
-              - "Makefile.debian9"
-              - "Makefile.debian10"
-              - "Makefile.debian11"
-              - "Makefile.debian12"
+        notify:
+          - github_commit_status:
+              context: "Release / Ubuntu ARM"
 
       - label: "Post-Release"
         key: "post-release"
         command: ".buildkite/scripts/post-release.sh ${GOLANG_VERSION}"
-        branches: "^(main|\\d+\\.\\d+)$"
+        if: build.branch == "main" || build.branch =~ /^[0-9]+\.[0-9]+$$/
+        depends_on:
```

Comment:
As far as I see this is not in the original PR -> https://github.com/elastic/golang-crossbuild/pull/330/files#diff-a3991ebf1475eb82acab13946ffc1eca02b43917f6d5bec3898f45c8b0b9bd53R100-R102